r-pkg/uptasticsearch/tests/testthat/test-repo_characteristics.R

```diff
@@ -39,7 +39,7 @@ test_that("R CMD check should not return any errors", {
 })
 
 test_that("R CMD check should not return any warnings", {
-    expect_true(length(x[["warnings"]]) == 0)
+    expect_true(length(x[["warnings"]]) <= 1)
```

Comment:
the test itself runs fine. I was checking using option `--as-cran --no-manual` and it pops a warning. Changed it back now.